assembl/views/discussion/views.py

```diff
@@ -215,6 +217,20 @@ def react_view(request, required_permission=P_READ):
     return react_base_view(request, required_permission)
 
 
+def get_assembl_version(discussion):
+    request = get_current_request()
+    user_id = request.authenticated_userid or Everyone
+    if discussion:
+        permissions = get_permissions(
+            user_id, discussion.id)
+        if P_ADMIN_DISC in permissions:
+            return pkg_resources.get_distribution("assembl").version
```

Comment:
After this line, you can just do `return ""` and remove the `else`

Reply:
done